src/Neon/Encoder.php

```diff
@@ -28,6 +28,13 @@ public function encode($var, $options = NULL)
 			return $var->format('Y-m-d H:i:s O');
 
 		} elseif ($var instanceof Entity) {
+			if($var->value === Neon::CHAIN) {
+				$s = '';
+				foreach($var->attributes as $attr) {
+					$s .= $this->encode($attr);
+				}
+				return $s;
```

Comment:
Will this work?

~~~php
return implode('', array_map([$this, 'encode'], $var->attributes));
~~~